Block/Script.php

```diff
@@ -149,6 +155,12 @@ public function _toHtml()
             ]));
         }
 
-        return $formatter->toScriptTag();
+        $nonce = null;
+        if (version_compare($this->getHelper()->getMagentoVersion(), '2.4.7', '>=')) {
```

Comment:
Do you think we could make a method for this to make it easier to use in the future? `$this->getHelper()->isMinimumMagentoVersion('2.4.7');`?

Reply:
Yeah I think that's a good idea. I've added that function.